### PR TITLE
Dependency issues fixes

### PR DESCRIPTION
**Pull request by Pessimistress**
- Remove `d3-request`, use `fetch` instead
- Remove non-browser dependency `path`
- Fix an invalid export
- Consistent naming for draco submodule (the code base uses `DRACO*` and `Draco*` 50-50 and the official documentation uses `Draco`)

### CODE REVIEW

File: modules/core/src/common/file-utils/loader.js [hunk 1]
```diff
@@ -1,43 +1,10 @@
-import {request as d3request} from 'd3-request';
-
+/* global fetch */
 export function loadFile(url, loader, options = {}) {
   if (loader.parseBinary) {
-    return loadBinary(url).then(data => loader.parseBinary(data, options));
+    return fetch(url).then(res => res.arrayBuffer()).then(data => loader.parseBinary(data, options));
```
Comment (ibgreen) [inline modules/core/src/common/file-utils/loader.js]: Nit: Move the conditional inside the `then`?

File: test/webpack.config.js [hunk 2]
```diff
@@ -89,7 +83,10 @@ const TEST_CONFIG = Object.assign({}, COMMON_CONFIG, {
   },
 
   plugins: [
-    new HtmlWebpackPlugin({title: 'loaders.gl tests'})
+    new HtmlWebpackPlugin({title: 'loaders.gl tests'}),
+    new webpack.DefinePlugin({
```
Comment (ibgreen) [inline test/webpack.config.js]: Where is `TEST_DATA_DIR` used? 

Node tests run without webpack, do they get access to this constant?

**Comment by Pessimistress** [comment]
Re: progress and abort

Looks like [response.body](https://developer.mozilla.org/en-US/docs/Web/API/Response) exposes some useful API for this. I'll add a TODO comment.

File: test/webpack.config.js (hunk 2 above)
Comment (Pessimistress) [reply test/webpack.config.js]: This was left over from something else I was testing. Will remove.

**Pull request merged**